CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.6.5
```

Comment:
Or do you all prefer to just suck in the changes and do the version bumping yourself?